pallets/order-book/src/lib.rs

```diff
@@ -448,58 +412,9 @@ pub mod pallet {
 			Self::fulfill_order_with_amount(order, amount_out, account_id)
 		}
 
-		/// Adds a valid trading pair.
-		#[pallet::call_index(4)]
-		#[pallet::weight(T::Weights::add_trading_pair())]
-		pub fn add_trading_pair(
-			origin: OriginFor<T>,
-			currency_in: T::CurrencyId,
-			currency_out: T::CurrencyId,
-			min_order: T::BalanceOut,
-		) -> DispatchResult {
-			T::AdminOrigin::ensure_origin(origin)?;
-
-			// We do not check, we just overwrite as this is an admin action.
-			TradingPair::<T>::insert(currency_in, currency_out, min_order);
-
-			Self::deposit_event(Event::<T>::TradingPairAdded {
-				currency_in,
-				currency_out,
-				min_order,
-			});
-
-			Ok(())
-		}
-
-		/// Removes a valid trading pair
-		//
-		// NOTE: We do not need to remove existing order as
-		//       fulfilling orders is not checking for a valid trading pair.
-		//       Existing orders will just fade out by by being canceled
-		//       or fulfilled.
-		#[pallet::call_index(5)]
-		#[pallet::weight(T::Weights::rm_trading_pair())]
-		pub fn rm_trading_pair(
-			origin: OriginFor<T>,
-			currency_in: T::CurrencyId,
-			currency_out: T::CurrencyId,
-		) -> DispatchResult {
-			T::AdminOrigin::ensure_origin(origin)?;
-
-			// We do not check, we just remove as this is an admin action.
-			TradingPair::<T>::remove(currency_in, currency_out);
-
-			Self::deposit_event(Event::<T>::TradingPairRemoved {
-				currency_in,
-				currency_out,
-			});
-
-			Ok(())
-		}
-
 		/// Set the market feeder for set market ratios.
 		/// The origin must be the admin origin.
-		#[pallet::call_index(6)]
+		#[pallet::call_index(4)]
```

Comment:
Not the biggest fan of changing call indexes. Just for the future. I would just deprecate old ones as external integrations "might" rely on the value.

Reply:
Gotcha.

> external integrations

Are we using external integrations for `order-book`? If not are we free to change them? I say this because in the previous PRs, a lot of extrinsic changed, maybe we should do some extra work related to it. 

If it doesn't matter this time for `order-book`, then I will leave it as it is to keep them simple and sorted.

Reply:
I am fine with changing